client-types/index.d.ts

```diff
@@ -0,0 +1,37 @@
+export {};
```

Comment:
I think that must not be a module to allow global declarations

Reply:
otherwise we have
```
TS2669: Augmentations for the global scope can only be directly nested in external modules or ambient module declarations.
```